CombinedDocument.md

```diff
@@ -907,6 +907,33 @@ extension ExampleViewController: UINavigationControllerDelegate {
 }
 ```
 
+# Formatting
+### How do we format code?
+Most formatting-related guidelines are enforced by [SwiftLint](https://github.com/realm/SwiftLint) using [our documented configuration file](https://github.com/Lickability/swift-best-practices/blob/master/.swiftlint.yml). Xcode’s default behaviors and preferences are preferred for considerations not covered by SwiftLint rules.
+
+### Key Considerations
+* Resolve any added SwiftLint warnings prior to opening a pull request with your changes.
```

Comment:
Should we talk about danger / setting up danger here?

Reply:
@Twigz I believe you had thoughts on whether or not to use danger in the future. Could you please weigh in here?

Reply:
We have no plans at this time, so no we shouldn’t talk about it.